websites/docs-smoke-test/src/content/index.mdx

```diff
@@ -29,3 +31,27 @@ title: Explore how to build a documentation website for commercetools
   <Card>Avengers: Age of Ultron</Card>
   <Card>Avengers: Infinity War</Card>
 </Cards>
+
+# Course cards
+
+<Cards narrow clickable>
+  <CourseCard duration="30 min" title="Course name" href="self-learning/overview" courseId="66">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+</Cards>
+
+# Learning path card
+
+<Cards>
+  <LearningPathCard title="Overview" duration="1 hour 25 minutes | 7 courses" productName="Composable Commerce">
+    This learning path is great for Composable Commerce administrators who create/maintain e-commerce data points, primarily work with a user interface, and have some familiarity with APIs.
+  </LearningPathCard>
+  <ImageCard>
```

Comment:
I thought to create a specific card just as image container, so that we can include it in the `Cards` component and get all the layout/responsivness features already implemented.

I haven't found a way to actually import the svg file into mdx so I tried to use the image syntax to load it as image and then wrap it into the `ImageCard` so that it can be positioned correctly.

Reply:
I'm running out of ideas but I',m open to any other solution @nkuehn

Reply:
Concerning the images: For right now whatever works is good.  

Take note of the problem separately, we'll need an author-friendly way to use SVGs in the MDX fiels as images that are not a content image.